lib/restrictedsession/restricted.go

```diff
@@ -298,6 +305,27 @@ func (l *auditEventLoop) close() {
 	l.wg.Wait()
 }
 
+// checkBpfLsm checks that eBPF is one of the enabled
+// LSM "modules".
+func checkBpfLsm() error {
+	const lsmInfo = "/sys/kernel/security/lsm"
+
+	csv, err := os.ReadFile(lsmInfo)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	for _, mod := range strings.Split(string(csv), ",") {
+		if mod == "bpf" {
+			return nil
+		}
+	}
```

Comment:
nit: we can do this without extra allocations

```suggestion
	for _, mod := range bytes.Split(csv, []byte(",")) {
		if bytes.Equal(mod, []byte("bpf")) {
			return nil
		}
	}
```